packages/backend/src/helpers/global-variable.ts

```diff
@@ -149,6 +152,10 @@ const globalVariable = async (
     return lastInternalIds?.includes(internalId)
   }
 
+  if (user) {
```

Comment:
nit: Actually why can't we store the entire user object? I got a bit irritated because I wanted to perform DB query on `$.flow`, and I realised It was `IFlow` and not `Flow`.

Reply:
I have no issues with this. Just thought it's better to store only what we need. Altho performance-wise will not matter since it's storing a pointer. We could extend this to store the entire user next time. No backwards compatibility issue since it is current used in a user action (verifyHook/registerHook) and not an action/trigger.